3574/2/helpers/save_helper.rb

```diff
@@ -0,0 +1,33 @@
+require 'fileutils'
+
+module SaveHelper
+  def save_data(action)
+    info = path_for(action)
+    make_dir(info)
+    save_photo(info)
+    save_location(info)
+  end
+
+  private
+
+  def make_dir(info)
+    session[:timestamp] = Time.now.strftime('%Y-%m-%d_%H-%M-%S')
+    FileUtils.mkdir_p(info)
+  end
+
+  def save_photo(info)
+    File.open(info + 'photo.jpg', 'wb') do |file|
```

Comment:
info + 'photo.jpg' - this is not clear, what is info? please choose better name for info variable